### PR TITLE
Add horti links, use  beta1 example of install

### PR DESCRIPTION
**Pull request by mrjones-plip**

### CODE REVIEW

**Comment by garethbowen** [comment]
@mrjones-plip Where is this PR at?

**Comment by mrjones-plip** [comment]
Thanks for the nudge @garethbowen ! I'd clearly forgotten about this - looks good to go (but you tell me ;)

File: content/en/core/releases/feature_releases.md [hunk 1]
```diff
@@ -19,14 +19,19 @@ A Feature Release can easily be identified by its version, which follows the pat
 
 ## Initial FR Installation
 
-When you are on a non-feature release, you need to use horticulturalist (horti) to do the initial install of the FR.  For example, if your instance was on `192-168-68-26.my.local-ip.co:8443` and you wanted to install `3.15.0-FR-bulk-user-upload-beta.1`, after installing `horti` you would could start the installation with this command:
+When you are on a non-feature release, you need to use horticulturalist (horti) to do the initial install of the FR.  For example, if your instance was runnint at `192-168-68-26.my.local-ip.co:8443` and you wanted to install `3.16.0-FR-offline-user-replace-beta.1`, after [installing](https://github.com/medic/horticulturalist#usage) `horti` you could start the installation with this command:
```
Comment (garethbowen) [inline content/en/core/releases/feature_releases.md:22]:
```suggestion
When you are on a non-feature release, you need to use horticulturalist (horti) to do the initial install of the FR.  For example, if your instance was running at `192-168-68-26.my.local-ip.co:8443` and you wanted to install `3.16.0-FR-offline-user-replace-beta.1`, after [installing](https://github.com/medic/horticulturalist#usage) `horti` you could start the installation with this command:
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
One typo suggestion inline. I'm not comfortable adding a new section with "TK" as the content - let's just leave that out until it's written?

Obviously we'll have to update this soon with 4.x info (eg: no horti) but that can be done in a separate PR.

File: content/en/core/releases/feature_releases.md [hunk 2]
```diff
@@ -19,14 +19,19 @@ A Feature Release can easily be identified by its version, which follows the pat
...
+COUCH_URL=https://medic:password@192-168-68-26.my.local-ip.co:8443/medic horti --local --install=3.16.0-FR-offline-user-replace-beta.1
 ```
 
 On subsequent upgrades to the later beta's of the FR, you will be able to more easily do it through the admin UI in the CHT.
 
-## Upgrades
+## Upgrade to next FR
 
-Once the feature is ready for widespread use it will be included in a regular CHT release. Projects using the feature version can be upgraded as soon as practical to get back on to a fully supported release.
+TK
```
[the remark this answers is not in the document]
Comment (mrjones-plip) [reply content/en/core/releases/feature_releases.md]: agreed! missed this when I sent it to you for review.  thanks for catching it!  Fixed in 504b25c0915cb04d6bfde04265ae2408e9342b09

**Review by garethbowen — APPROVED** [review]

**Pull request merged**